guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+offers a bit more:
+
+### Permissionless collaboration
+
+Anyone can propose changes. You don't need to be given permission to view or edit
+a document, as is required by some of the platforms mentioned previously.
+
+### More advanced reviewing tools
+
+When you open up a Google Doc, you might be able to comment or write in revision mode, but
+when you do so, you are working with the entire document. GitHub allows
+review on the level of change. Along with this more detailed ability to edit,
+there is an integrated approval workflow, straightforward visualizations of what changed 
+(these are called "diffs"), and powerful automatic testing (called continuous integration or CI).
+
+### More detailed history
+
+You are going to have basic questions like:
+
+ - Who added this piece?
+ - When and why was this added?
+
+And GitHub has built-in tools to do this (one such action is called [git blame](https://help.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file).)
+
+### Easy for many people to collaborate at once
+
+You can work in your own "alternate reality" without interfering with other work, even simultaneously. When ready these alternatives can be merged back together.
+You can also work on changes out of order, although this is more of an advanced topic.
+
+### Web Interfaces
+
+By way of some helpful features, the text files that you are working on can be served as web pages
+directly from where they live. You can read about this feature, called [GitHub Pages](https://pages.github.com/).
+
+To summarize and answer this question, we use GitHub because we want to:
+
+ 1. easily collaborate on common files
+ 2. have advanced reviewing tools and version history
+ 3. keep a record of discussions, and every change made
+ 4. be able to revert back to previous versions, if needed
+ 5. render some of the static files in a web interface
+
+
+## What content do we have on GitHub?
+
+Generally, anything that you'd want to put in a digital safe deposit box is what you'll
+find on GitHub. This obviously includes code, but also includes documentation and
+other web content. Specifically, the [sourcecred organization](https://github.com/sourcecred)
+has the following sets of files. Each set is called a "repository" and you can think of
+it like a (sub)-project under a GitHub organization, or a GitHub username. Each repository
+has it's own link and space on the GitHub website.
+
+ - [sourcecred](https://github.com/sourcecred/sourcecred): the core sourcecred codebase.
+ - [docs](https://github.com/sourcecred/docs): is a growing set of documentation and community health files. In fact, the file that you are reading right now is part of it!
+ - [sourcecred-action](https://github.com/sourcecred/sourcecred-action): a set of workflow that can be used by other GitHub communities.
+
+If you are familiar with filesystems and permissions, you can think of each
+repository as a folder or space that carries it's own permissions, labels,
```

Comment:
its* own permissions (no apostrophe)

Reply:
Looks like I had another below that too! Oups :)